src/utils/PdfGenerator/HBSCompiler.ts

```diff
@@ -0,0 +1,14 @@
+import * as hbs from "handlebars"
+import fs from "fs-extra"
+import * as path from "path"
```

Comment:
czy tak nie zadziała?
```suggestion
import path from "path"
```